x-pack/plugins/synthetics/server/routes/monitor_cruds/add_monitor.ts

```diff
@@ -79,6 +77,7 @@ export const addSyntheticsMonitorRoute: SyntheticsRestApiRouteFactory = () => ({
     );
 
     try {
+      const { id: spaceId } = await server.spaces.spacesService.getActiveSpace(request);
```

Comment:
Because this route (and `delete_monitor` and `edit_monitor`) is internal, this is not as important for these routes and might add additional overhead unnecessarily. I don't have too strong of an opinion, but using this method is only strictly necessary for project monitors imo.

Reply:
I think we can keep it as a defense against people messing with the internal URLs.